agent/custom/bbc_connection_manager.py

```
@@ -88,6 +141,30 @@ def _start_permanent_listener(self):
         except Exception as e:
             mfaalog.error(f"[BbcConnectionManager] 启动永久监听失败: {e}")
     
+    def _restart_callback_listener(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_restart_callback_listener`

Add return type annotation: `None`

(ANN202)